legal-api/src/legal_api/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.97.0'  # pylint: disable=invalid-name
+__version__ = '2.98.0'  # pylint: disable=invalid-name
```

Comment:
Don't need to increment the version number for BE.  

The release champion can take care of this right before pushing to Test on the next release cycle.

Reply:
okay

Reply:
fixed